lib/MusicBrainz/Server/Entity/URL/IMSLP.pm

```diff
@@ -44,7 +38,7 @@ IMSLP URLs are only show in the sidebar if the URL can be decoded from utf-8
 
 =cut
 
-sub show_in_sidebar { defined(shift->utf8_decoded) }
+sub show_in_sidebar { !shift->uses_legacy_encoding }
```

Comment:
Interestingly, the string value of the old `utf8_decoded` attribute was used nowhere outside the base class; all subclasses only checked for definedness (was undefined for legacy-encoded URLs) and otherwise did their own decoding.